### PR TITLE
samples readme

### PR DESCRIPTION
**Pull request by alemata**

### CODE REVIEW

**Comment by alemata** [comment]
@flaviogranero could you review please?

File: 07-Authorization/README.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
-# Authorization example.
+# Authorization
+
+This example shows one of the ways of adding ***Authorization*** for a resource in your application. We will have an `/admin` page, which will only be accessible for users with an `admin` role.
```
Comment (flaviogranero) [inline 07-Authorization/README.md]: @alemata I think we should use present here: We have an `/admin` page, which is only accessible for users with an `admin` role.

File: 09-MFA/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# MFA
+
+This example shows how to add ***MultiFactor Authentication*** to your `Auth0` authentication flow. You just need to config your auth0 account to add MFA and then make a login. There is no need of extra code configuration. 
+
```
Comment (flaviogranero) [inline 09-MFA/README.md]: the quickstart link is missing

**Comment by flaviogranero** [comment]
👍 @chenkie

**Pull request merged**